optimum/habana/transformers/models/mixtral/modeling_mixtral.py

```diff
@@ -150,6 +148,64 @@ def gaudi_mixtral_repeat_kv(
     return query_states, key_states, value_states, attention_mask
 
 
+class GaudiMixtralSparseMoeBlock(MixtralSparseMoeBlock):
+    def __init__(self, config):
+        super().__init__(config)
+
+    def forward(self, hidden_states: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
+        batch_size, sequence_length, hidden_dim = hidden_states.shape
+        original_shape = hidden_states.shape
```

Comment:
```suggestion
        original_shape = hidden_states.shape
        hidden_dim = original_shape[2]
```

We do not need `batch_size` & `sequence_length`